terraso_backend/apps/auth/views.py

```diff
@@ -72,19 +74,51 @@ def process_callback(self):
         response = HttpResponseRedirect(f"{settings.WEB_CLIENT_URL}/{self.state}")
         response.set_cookie("atoken", access_token, domain=settings.AUTH_COOKIE_DOMAIN)
         response.set_cookie("rtoken", refresh_token, domain=settings.AUTH_COOKIE_DOMAIN)
+        if created_with_service:
+            # Set samesite to avoid warnings
+            plausible_service = PlausibleService()
+            try:
+                plausible_service.track_signup(created_with_service, self.request)
+            except httpx.HTTPStatusError as e:
+                logger.error(
+                    "Error tracking signup: received status code %s when querying %s",
+                    e.response.status_code,
+                    e.request.url,
+                )
+            except Exception:
+                logger.exception("Error tracking signup")
```

Comment:
```suggestion
            except Exception as e:
                logger.exception("Error tracking signup: " +str(e))
```

Reply:
question: would it make sense to include details about the exception

Reply:
Apparently the Python `logger.exception` does this already ([documentation](https://docs.python.org/3/library/logging.html#logging.Logger.exception)). I learned this doing this change actually.